projects/replicated-rbac/rules.yaml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Is this rule necessary? I would think this is enforced by `required: ["name" ...]` in the schema

Reply:
because we are just verifying shape of the policy i don't think so that we need rules.yaml file. but when i'm trying to remove that i'm getting some error. so i need to figure out what to do.

Reply:
You might be able to just set rules.yaml to an empty list:

```
---
[]
```